tests/ellipsoidal_chunk_coordinate_parallel.prm

```diff
@@ -0,0 +1,91 @@
+# A simple setup for convection in a 3d ellipsoidal chunk. See the
+# manual for more information.
+
+############### Global parameters
+# We use a 3d setup. Since we are only interested
+# the shape of the grid, we set the end time
+# equal to the start time to force a single time
+# step before the program terminates.
+
+set Dimension                              = 3
+set Use years in output instead of seconds = true
+set End time                               = 0
+
+
+############### Parameters describing the model
+# We create a coordinate parallel ellipsoidal chunk with
+# a large eccentricity (0.5).
+
+subsection Geometry model
+  set Model name = ellipsoidal chunk
+    subsection Ellipsoidal chunk
+        set NE corner = -10:30
+		set SW corner = 40:-10
+		set Depth = 1000000
+		set Eccentricity = 0.5
+        set East-West subdivision = 4
+        set North-South subdivision = 4
+        set Depth subdivision = 1
```

Comment:
The output for this testcase is pretty large. Can you use, for example, 2/3/1 subdivisions? This way you can also verify that you've got the E-W and N-S subdivisions right (because you choose different numbers of subdividions for these two parameters).